code/game/gamemodes/clock_cult/clock_items.dm

```diff
@@ -253,40 +253,51 @@
+		capacitors, replicant alloy, and hierophant ansibles. Scripture needs different types of components to function, and usually consumes at least a few of those. In general, eyes are \
+		offensive, cogs are defensive, capacitors are for intelligence or mind control, alloys are for making stuff, and ansibles are for transmitting stuff. Simple enough, but important!<br><br>\
 		\
-		The third to fifth functionalities are several buttons in the top left while holding the slab, from left to right, they are:<br>\
-		<b><font color=#DAAA18>Hierophant Network</font></b>, which allows communication to other servants.<br>\
-		<b><font color=#AF0AAF>Guvax</font></b>, which simply allows you to quickly invoke the Guvax scripture.<br>\
-		<b><font color=#1E8CE1>Vanguard</font></b>, which, like Guvax, simply allows you to quickly invoke the Vanguard scripture.<br><br>\
+		Oh, another thing to note: scripture is divided into five <b>tiers</b>. Y'see, as Ratvar's presence grows on this deathtrap you call a station, all of his servants grow more resistant to \
+		the power of the scriptures. Although technically I could let you make the Ark right now, your head would probably explode from the mental strain, and I think you like having one. For \
+		your sake, we keep them tiered up, with scripture becoming steadily more powerful depending on tier. You only have the Driver tier unlocked by default, and it has stuff that's basic but \
+		important. As we grow more powerful, you unlock further tiers. I'll detail the requirements down below.<br><br>\
 		\
-		Examine the slab for component amount information.<br><br>\
+		Anyway, that's about all you need to kn- actually, wait. Something you should know. Unlike Nar-Sie's cult, which focuses on sabotage and doesn't really stay anywhere for long, we're much \
+		more static. That means that, instead of staying on the move, we make fortified strongholds and set up our operations there. Nezbere, Ratvar's engineer, has designed a lot of structures, \
+		so we need somewhere to put them. The most important of these is the <b>tinkerer's cache</b>, which lets you store components for later use. You can offload my (the slab) components into \
+		one by hitting it with me. Caches are so good because <i>everyone</i> can draw from the components inside, all caches are linked, and the components can be used to power scripture! It's \
+		a wonderful thing, teamwork. Anyway, the cache and other structures have something called <b>CV</b> - that stands for Construction Value. When you see CV mentioned in a tier unlock, we \
+		need that much <i>combined</i> CV to unlock it.<br><br>\
 		\
-		A complete list of scripture, its effects, and its requirements can be found below.<br>\
+		Ol gur Nex, listen to me ramble. You probably just want to know the scripture, so let me tell you. I can't teach you any more, but feel free to ask your fellow servants for help. Anyway, \
+		here's your scripture...<br>\
```

Comment:
I'm aware, but it's also more informative. I feel like, in the long run, it'll help more.